system-int-test/test/cmr/system_int_test/search/service/service_search_test.clj

```diff
@@ -576,15 +576,19 @@
 (deftest service-search-sort
   (let [service1 (services/ingest-service-with-attrs {:native-id "svc1"
                                                       :Name "service"
+                                                      :LongName "LongName1"
```

Comment:
Can you update the long names so that the order is something other than service 1, 2, 3, 4? That way it's clearer that the sort is working by long name instead of some default sort order.

Reply:
Done.